ui/app/components/keyboard-shortcuts-modal.js

```diff
@@ -42,18 +43,25 @@ export default class KeyboardShortcutsModalComponent extends Component {
     }
   }
 
-  tetherToElement(self, _, { element, hint }) {
-    let binder = new Tether({
-      element: self,
-      target: element,
-      attachment: 'top left',
-      targetAttachment: 'top left',
-      targetModifier: 'visible',
-    });
-    hint.binder = binder;
+  @action
+  tetherToElement(element, hint, self) {
+    if (!this.config.isTest) {
```

Comment:
Tried very hard to avoid using a config.isTest call, but Tether doesn't play nice with Ember tests at all. A simple check here solves a world of headaches.